app/views/notifications/index.html.erb

```diff
@@ -18,14 +18,27 @@
 </div>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <%= form_with url:notifications_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
+    <%= form_for @user_notification_preferences,
```

Comment:
@aliuk2012 is the form supposed to have a breadcrumb, back or cancel button?

Reply:
I do recall there being some form of navigation. Let me me do some digging around and I'll get back to you.

Reply:
We do need a "Cancel link" at the bottom of the form to take the user back to the previous page.

https://publish-courses-pr-44.herokuapp.com/notifications